docs/system-reqs.md

```diff
@@ -3,24 +3,24 @@
 The v1.0 or later version of 3PAR Volume plugin of Docker Engine Managed plugin system is supported and available now on Docker store (https://store.docker.com/plugins/hpe-3par-docker-volume-plugin/).
 Versions starting at v1.0 have been tested and are supported on the following Linux distributions:
 
-- Ubuntu 16.04
```

Comment:
I would suggest not to keep later word until unless we qualify and certify the OS

Reply:
@prablr79 , I agree and we should'nt claim for Operating system version which are untested.

Reply:
Then we need to keep this updated on exact OS versions we support. Please send me that list.

Reply:
@budhac its important that we don't put "latest" word until unless we qualify our work with respect to the supported OS.  Lets keep updating this information going forward.